src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -406,186 +491,23 @@ describe('StripeV3PaymentStrategy', () => {
             jest.spyOn(store.getState().shippingAddress, 'getShippingAddress').mockReturnValue(undefined);
             jest.spyOn(store.getState().billingAddress, 'getBillingAddress').mockReturnValue(undefined);
 
-            await strategy.initialize(stripeV3Options);
-            const response = await strategy.execute(getStripeV3OrderRequestBodyMock());
+            await strategy.initialize(options);
+            const promise = await strategy.execute(getStripeV3OrderRequestBodyMock());
```

Comment:
`response` is more appropriate naming, as you are using `await`. The const will hold the response, not the promise.